### ISSUE TITLE
Replace dropdown component

### ISSUE THREAD

**Issue by calebissharp** [bcgov/range-web#547]
We've found the semantic-ui dropdown doesn't work for well for our use-cases. There are also some long-standing bugs that cause strange behaviour. This issue only addresses dropdowns being used as selects. 

A possible replacement is to use the dropdown from [material-ui](https://material-ui.com/components/selects/#select) or [react-select](https://github.com/JedWatson/react-select)

See #525 and #164 for required behaviour.

**Comment by LisaMoore1** [comment]
@calebissharp is this a separate issue from what you recently worked on (#532)?

**Comment by calebissharp** [comment]
@LisaMoore1 Correct. This issue tracks replacing *every* dropdown. So far I've just replaced the ones in the grazing schedules to resolves #532. I assume replacing the rest are a lower priority?

**Comment by LisaMoore1** [comment]
Yup. All good where it is just wanted to understand.
